README.md

```diff
@@ -20,4 +20,7 @@ Reverse engineer binary files
 ## Start the project
 ` yarn start ` or `npm start`
```

Comment:
We need to run `yarn build` now first, right?  `yarn start` doesn't run `tsc` any more.